extensions/amp-story/1.0/amp-story.js

```diff
@@ -347,6 +350,7 @@ export class AmpStory extends AMP.BaseElement {
       this.analytics_.onNavigationStateChange(stateChangeEvent);
     });
 
+    this.extractTitle_();
```

Comment:
If we don't need to store the title, could we just have a `this.element.removeAttribute('title')` here?
Maybe it's worth adding a super small comment and link the github issue?

Reply:
I would also be okay with this, then the follow-up PR to actually propagate the 1.0 metadata though to the bookend etc can worry about storing it.